pkg/gits/operations/pull_request_op_test.go

```diff
@@ -103,6 +103,87 @@ func TestCreatePullRequest(t *testing.T) {
 		results.PullRequestArguments.Title, "The PR title should contain the old and new versions")
 }
 
+func TestCreatePullRequestsWithLabels(t *testing.T) {
+
+	_, _, _, commonOpts, _ := getFakeClientsAndNs(t)
+
+	testOrgName := "testowner"
+	testRepoName := "testrepo"
+
+	commonOpts.SetGit(gits.NewGitFake())
+
+	gitter := gits_test.NewMockGitter()
+
+	fakeRepo, _ := gits.NewFakeRepository(testOrgName, testRepoName, nil, nil)
+	fakeGitProvider := gits.NewFakeProvider(fakeRepo)
+	fakeGitProvider.User.Username = testOrgName
+
+	o := operations.PullRequestOperation{
+		CommonOptions: &commonOpts,
+	}
+
+	testhelpers.ConfigureTestOptionsWithResources(o.CommonOptions,
+		[]runtime.Object{},
+		[]runtime.Object{},
+		gitter,
+		fakeGitProvider,
+		nil,
+		resources_test.NewMockInstaller(),
+	)
+
+	err := testhelpers.CreateTestEnvironmentDir(o.CommonOptions)
+	assert.NoError(t, err)
+
+	pegomock.When(gitter.HasChanges(pegomock.AnyString())).ThenReturn(true, nil)
+
+	o.GitURLs = []string{"testowner/testrepo"}
+	o.SrcGitURL = "testowner/testrepo"
+	o.Version = "3.0.0"
+
+	var results *gits.PullRequestInfo
+
+	logOutput := log.CaptureOutput(func() {
```

Comment:
I don't think asserting against user output makes for good tests. That's fragile at best.
Would you not go via the _fakeGitProvider_, go get to the appropriate fake repository and assert that there was a (fake) PR added and that this PR has labels set? I think that is a much more direct test.